federation/cmd/federation-apiserver/app/core.go

```diff
@@ -33,20 +33,21 @@ import (
 	"k8s.io/kubernetes/pkg/api/rest"
 	"k8s.io/kubernetes/pkg/apimachinery/registered"
 	"k8s.io/kubernetes/pkg/genericapiserver"
-	configmapetcd "k8s.io/kubernetes/pkg/registry/core/configmap/etcd"
-	eventetcd "k8s.io/kubernetes/pkg/registry/core/event/etcd"
-	namespaceetcd "k8s.io/kubernetes/pkg/registry/core/namespace/etcd"
-	secretetcd "k8s.io/kubernetes/pkg/registry/core/secret/etcd"
-	serviceetcd "k8s.io/kubernetes/pkg/registry/core/service/etcd"
+	configmapstore "k8s.io/kubernetes/pkg/registry/core/configmap/storage"
```

Comment:
Or alternatively, we can change the directory to "store" instead, but I think that option is worse.